pkg/provisioners/gcppubsub/controller/channel/reconcile.go

```diff
@@ -140,20 +151,34 @@ func (r *reconciler) reconcile(ctx context.Context, c *eventingv1alpha1.Channel)
 	// 3. The GCP PubSub Topic (one for the Channel).
 	// 4. The GCP PubSub Subscriptions (one for each Subscriber of the Channel).
 
+	// First we will plan all the names out for steps 3 and 4 persist them to status.raw. Then, on a
+	// subsequent reconcile, we manipulate all the GCP resources in steps 3 and 4.
+
 	// Regardless of what we are going to do, we need GCP credentials to do it.
 	gcpCreds, err := pubsubutil.GetCredentials(ctx, r.client, r.defaultSecret, r.defaultSecretKey)
 	if err != nil {
 		logging.FromContext(ctx).Info("Unable to generate GCP creds", zap.Error(err))
 		return false, err
 	}
 
+	originalPBS, err := pubsubutil.ReadRawStatus(ctx, c)
```

Comment:
:thinking: Let's try to come up with a better name for this than PBS

Reply:
Switched to `pcs`.